Formula/cp2k.rb

```diff
@@ -0,0 +1,65 @@
+class Cp2k < Formula
+  desc "Quantum chemistry and solid state physics software package"
+  homepage "https://www.cp2k.org"
+  url "https://downloads.sourceforge.net/project/cp2k/cp2k-5.1.tar.bz2"
+  sha256 "e23613b593354fa82e0b8410e17d94c607a0b8c6d9b5d843528403ab09904412"
+
+  depends_on :fortran
+  depends_on :mpi => [:cc, :cxx, :f77, :f90]
+  depends_on "fftw"
+  depends_on "gcc"
+  depends_on "libxc"
+  depends_on "scalapack"
+
+  needs :openmp
+
+  resource "libint" do
+    url "https://downloads.sourceforge.net/project/libint/v1-releases/libint-1.1.5.tar.gz"
+    sha256 "31d7dd553c7b1a773863fcddc15ba9358bdcc58f5962c9fcee1cd24f309c4198"
+  end
+
+  def install
+    resource("libint").stage do
+      system "./configure", "--prefix=#{prefix}/dependencies"
+      system "make"
+      system "make", "install"
+    end
+
+    fcflags = "-I#{Formula["libxc"].opt_prefix}/include -I#{Formula["fftw"].opt_prefix}/include -I#{prefix}/dependencies/include"
+    libs = "-L#{Formula["libxc"].opt_prefix}/lib -lxcf90 -lxc -L#{prefix}/dependencies/lib -lderiv -lint -L#{Formula["fftw"].opt_prefix}/lib -lfftw3"
+
+    # CP2K configuration is done through editing of arch files
+    inreplace Dir["arch/Darwin-IntelMacintosh-gfortran.*"].each do |s|
+      s.gsub! /DFLAGS *=/, "DFLAGS = -D__LIBXC -D__FFTW3 -D__LIBINT"
```

Comment:
can we use `change_make_var!`?

Reply:
Nope, because we want to preserve the existing content.